letsencrypt/le_util.py

```diff
@@ -255,3 +257,26 @@ def safe_email(email):
     else:
         logger.warn("Invalid email address: %s.", email)
         return False
+
+
+def add_deprecated_argument(add_argument, argument_name, nargs):
```

Comment:
I'd argue that it should not go in `HelpfulArgumentParser` as then you'd need access to the parser object to be able to call the function. This approach gives 3rd party plugins a function they can use to deprecate their own arguments.

If we split `cli.py` into `main.py` and `cli.py` as you have suggested, I think putting it in `cli.py` makes sense. I personally like it best in `le_util.py` based on the current structure though.